topology/University of Michigan/AGLT2/AGLT2.yaml

```
@@ -250,7 +250,9 @@ Resources:
           Name: Shawn McKee
       Security Contact:
         Primary:
-          ID: 737e25f3cb6a16c6c1846ee2601b5e5b102bc157
+          ID: 737e25f3cb6a16c6c1846ee2601b5e5b
+          
+          2bc157
```

```suggestion
          ID: 737e25f3cb6a16c6c1846ee2601b5e5b2bc157
```

@wuwj6269 this change looks unintended